site/content/docs/v1.9.0/annotations.md

```diff
@@ -23,7 +23,7 @@ In both cases, they will behave as follows, by default:
 * If specified as `kubernetes.io/ingress.class: contour`, then Contour serves the Ingress.
 * If any other value, Contour ignores the Ingress definition.
 
-You can override the default class `contour` by providing the `--ingress-class-name` flag to Contour. 
+You can override the default class `contour` by providing the `--ingress-class-name` flag to Contour.
```

Comment:
can probably get rid of this change being part of this PR too, since this is quite an old version etc.